CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Bugs fixed:
 * Override purchase order lines limit setting field allows saving fractional numbers (UIDATIMP-1383)
 * Order field mapping profile: Fix the values in the Payment status field (UIDATIMP-1386)
+* Manual clearing of the Vendor/Material supplier/Access provider fields is not working properly (UIDATIMP-1388)
```

Comment:
Hi @Terala-Priyanka! Since the correct sequence of tickets in the list is not described anywhere, we decided to add tickets in ascending order for `data-import`. Correct me please if I'm wrong.

Reply:
The latest implementation should fall at the end of the list under "in progress".
From your change log file, it likes like you have to swap "* Order import: creating an order with Receipt not required causes an error (UIDATIMP-1389)" with "* Manual clearing of the Vendor/Material supplier/Access provider fields is not working properly (UIDATIMP-1388)"

Reply:
Hi @Terala-Priyanka In the data import module our team decided to organize tickets in the CHANGELOG by ticket number in ascending order, it helps us during releases
1. to check if all needed tickets were included in a release and that we didn't miss anything
2. sort and check with Jira tickets
Since the CHANGELOG is used primarily to list updates to a particular release, I don't see any difference in the order in which the tickets are listed. But maybe we have some rules regarding this and I just missed it?